src/KIM/kim_init.cpp

```diff
@@ -168,6 +168,56 @@ void get_kim_unit_names(
 }
 }  // namespace
 
+void KimInit::print_dirs(struct KIM_Collections * const collections) const
+{
+  int kim_error = 0;
+  int dirListExtent=0;
+  input->write_echo("#=== KIM is looking for 'Portable Models' in these directories ===\n");
+  std::vector<struct KIM_Collection> collection_list;
+  collection_list.push_back(KIM_COLLECTION_currentWorkingDirectory);
+  collection_list.push_back(KIM_COLLECTION_environmentVariable);
+  collection_list.push_back(KIM_COLLECTION_user);
+  collection_list.push_back(KIM_COLLECTION_system);
+
+  for (auto col=collection_list.begin(); col != collection_list.end(); ++col)
```

Comment:
Since we now require C++11, couldn't this be further simplified to: `for (auto col : collection_list) {`?